task1/task1/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("В листопаде, яркое");
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine(@"В листопаде, яркое,
+Что за чудо, спросим? -
+На дары богатое
+Время года осень.");
+            Console.ForegroundColor = ConsoleColor.White;
```

Comment:
Лучше восстановить цвета по умолчанию консоли пользователя, а не ваши.
```suggestion
            Console.ResetColor();
```